.github/workflows/coverage.yml

```diff
@@ -24,13 +24,13 @@ jobs:
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
-          php-version: 8.1
```

Comment:
Any reason to bump to PHP 8.2? I see PHP 8.1 is the minimum version supported by L10. Wouldn't it be safer to use the minimum version, just in case?

Reply:
This is just the PHP version used to generate the coverge, since I use PHP 8.2 locally, and usually I am the one pushing tests ... I just updated it to be conformant with my local setup. 

We can revert, no issue.